src/core/PlaneWaveBasis.jl

```diff
@@ -0,0 +1,216 @@
+using FFTW
+
+@doc raw"""
```

Comment:
I think this is unnecessary?

Reply:
I think it's needed because oft the LaTeX (See Documenter.jl examples), but I will check.

Reply:
It is indeed required.